### PR TITLE
Feature contact form

### PR DESCRIPTION
**Pull request by Tchilo**
Add form to html
Add style for form in css

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -184,11 +184,12 @@ <h2 class="me">About me</h2>
       <h3>I'm always interested in 
         hearing about new projects, so if you'd like to chat please get in touch.</h3>
       <form action="https://formspree.io/f/xzbygvjd" method="POST">
-        <input class="fname" type="text" maxlength="30" value="Full name" name="full-name" placeholder="Full name">
-        <input class="email" type="email" value="email" name="email-address" placeholder="Email address">
-        <textarea class="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="enter text here"></textarea>
+        <input class="fname" type="text" maxlength="30"  name="full-name" placeholder="Full name">
+        <input class="email" type="email" name="email-address" placeholder="Email address">
+        <textarea class="textarea" name="comment" maxlength="500" cols="30" rows="10" placeholder="Enter text here"></textarea>
```
Comment (mricanho) [inline index.html:189]: - [ ] All input fields should have the required attribute to activate the validation on submit.

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @Tchilo @pasytchangwa, 

Your contact form is really good, you are following the design, the form is successfully connected to FormSpree and your validations are mostly there.

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

this close

You are so close, but it's a quick change, you can do it!

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Tchilo) [reply index.html]: Done

**Review by Shaher-11 — APPROVED** [review]
Hello @Tchilo ,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle: 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**